frontend/src/components/widgets/Checkbox/Checkbox.tsx

```diff
@@ -38,7 +38,7 @@ interface State {
 
 class Checkbox extends React.PureComponent<Props, State> {
   public state: State = {
-    value: this.props.element.get("default"),
+    value: this.props.element.get("value"),
```

Comment:
Why rename the field?

Reply:
IMHO I think the change makes sense; "default" and "value" convey two different senses of what the number inside the variable represents.